### PR TITLE
Update Japanese translation (LINE 764-812)

### PR DESCRIPTION
**Pull request by yosshy**
/hold for japanese developer's review
/assign @atoato88
/cc @YuikoTakada @t-inu @s-kawamura-w664

### CODE REVIEW

**Comment by yosshy** [comment]
I signed it

**Comment by yosshy** [comment]
I signed it

File: translations/ja.json [hunk 1]
```diff
@@ -761,55 +761,55 @@
...
-	"Unable to remove machine directory": "",
-	"Unable to restart cluster, will reset it: {{.error}}": "",
-	"Unable to safely downgrade existing Kubernetes v{{.old}} cluster to v{{.new}}": "",
-	"Unable to stop VM": "",
-	"Unable to update {{.driver}} driver: {{.error}}": "",
-	"Unfortunately, could not download the base image {{.image_name}} ": "",
+	"Unable to push cached images: {{.error}}": "キャッシュされたイメージを登録できません: {{.error}}",
+	"Unable to remove machine directory": "マシンディレクトリーを削除できません",
+	"Unable to restart cluster, will reset it: {{.error}}": "クラスターを再起動できません (リセットします): {{.error}}",
+	"Unable to safely downgrade existing Kubernetes v{{.old}} cluster to v{{.new}}": "既存の Kubernetes v{{.old} クラスターを v{{.new}} に安全にバージョンダウンできません",
```
Comment (spowelljr) [inline translations/ja.json:767]:
```suggestion
	"Unable to safely downgrade existing Kubernetes v{{.old}} cluster to v{{.new}}": "既存の Kubernetes v{{.old}} クラスターを v{{.new}} に安全にバージョンダウンできません",
```

**Review by spowelljr — CHANGES_REQUESTED** [review]

File: translations/ja.json [hunk 2]
```diff
@@ -761,55 +761,55 @@
...
+	"Usage: minikube completion SHELL": "使用法: minikube completion SHELL",
+	"Usage: minikube delete": "使用法: minikube delete",
+	"Usage: minikube delete --all --purge": "使用法: minikube delete --all --purge",
+	"Usage: minikube node [add|start|stop|delete|list]": "使用法: minikube node [add|start|stop|delete|list]",
+	"Usage: minikube node delete [name]": "使用法: minikube node delete [ノード名]",
+	"Usage: minikube node list": "使用法: minikube node list",
+	"Usage: minikube node start [name]": "使用法: minikube node start [ノード名]",
+	"Usage: minikube node stop [name]": "使用法: minikube node stop [ノード名]",
+	"Use \"{{.CommandPath}} [command] --help\" for more information about a command.": "コマンドに関する追加情報は「{{.CommandPath}} [command] --help」を使用してください。",
+	"Use 'kubect get po -A' to find the correct and namespace name": "'kubect get po -A' を使用して、妥当なネームスペース名を見つけてください",
```
Comment (atoato88) [inline translations/ja.json:796]: Just comment.
May it be typo in original text? :thinking:

File: translations/ja.json [hunk 3]
```diff
@@ -761,55 +761,55 @@
...
+	"Use -A to specify all namespaces": "-A を使用して、全ネームスペースを指定してください",
+	"Use SSH connection instead of HTTPS (port 2376)": "HTTPS 接続の代わりに SSH 接続を使用してください (ポート 2376)",
+	"Use SSH for running kubernetes client on the node": " SSH を使用して、ノード上の Kubernetes クライアントを実行ください",
+	"Use VirtualBox to remove the conflicting VM and/or network interfaces": "VirtualBox を使用して、衝突した VM やネットワークインターフェイスを削除してください",
+	"Use native Golang SSH client (default true). Set to 'false' to use the command line 'ssh' command when accessing the docker machine. Useful for the machine drivers when they will not start with 'Waiting for SSH'.": "ネイティブの Go 言語 SSH クライアントを使用してください (デフォルトは true)。Docker マシンにアクセスする際には、'false' をセットして、コマンドラインの 'ssh' コマンドを使用してください。マシンドライバーが 'Waiting for SSH' で開始されない場合に有用です。",
```
Comment (atoato88) [inline translations/ja.json:801]: `--native-ssh` option on [here](https://minikube.sigs.k8s.io/docs/commands/ssh/#options) is the place?
If so, it's better I think.
 ```suggestion
	"Use native Golang SSH client (default true). Set to 'false' to use the command line 'ssh' command when accessing the docker machine. Useful for the machine drivers when they will not start with 'Waiting for SSH'.": "ネイティブの Go 言語 SSH クライアントを使用します (デフォルトは true)。Docker マシンにアクセスする際に、コマンドラインの 'ssh' コマンドを使用する場合は 'false' をセットしてください。マシンドライバーが 'Waiting for SSH' で開始されない場合に有用です。",
```

Comment (atoato88) [reply translations/ja.json:801]: Oh, it does not looks the exact text. but meaning is same with this text, right?

**Comment by atoato88** [comment]
I left some comments. Could you check this?
1 review comment and 1 comment.

File: translations/ja.json (hunk 1 above)
Comment (yosshy) [reply translations/ja.json:767]: @spowelljr Thank you for your review! Tell me how to check such a mistake if exists.

File: translations/ja.json [hunk 4]
```diff
@@ -761,55 +761,55 @@
...
+	"Unpaused {{.count}} containers in: {{.namespaces}}": "次のネームスペースに存在する {{.count}} 個のコンテナーを再稼働させました: {{.namespaces}}",
+	"Unpausing node {{.name}} ...": "{{.name}} ノードを再稼働させています ...",
+	"Unpausing node {{.name}} ... ": "{{.name}} ノードを再稼働させています ... ",
+	"Unset the KUBECONFIG environment variable, or verify that it does not point to an empty or otherwise invalid path": "環境変数 KUBECONFIG をセット解除するか、同変数が空または不正なパスに設定されていないことを確認してください",
+	"Unset variables instead of setting them": "変数をセットする代わりに、セット解除してください",
+	"Update Docker to the latest minor version, this version is unsupported": "Docker を最新のマイナーバージョンに更新してください (このバージョンは未サポートです)",
+	"Update kubeconfig in case of an IP or port change": "IP アドレスやポート番号が変わった場合に kubeconfig を更新してください",
+	"Update server returned an empty list": "空リストを返したサーバーを更新してください",
+	"Updating the running {{.driver_name}} \"{{.cluster}}\" {{.machine_type}} ...": "実行中の {{.driver_name}} 「{{.cluster}}」 {{.machine_type}} を更新しています...",
+	"Upgrade to QEMU v3.1.0+, run 'virt-host-validate', or ensure that you are not running in a nested VM environment.": "QEMU v3.1.0 以降にアップグレードするか、'virt-host-validate' を実行するか、ネストされた VM 環境中で実行されていない事を確認してください。",
```
[the remark this answers is not in the document]
Comment (yosshy) [reply translations/ja.json:784]: チェック漏れでした。指摘ありがとうございます。

File: translations/ja.json (hunk 2 above)
Comment (yosshy) [reply translations/ja.json:796]: @atoato88 thanks for your comment. It looks so. I'll make an issue to fix typos including it.

File: translations/ja.json [hunk 5]
```diff
@@ -761,55 +761,55 @@
...
+	"User name '{{.username}}' is not valid": "ユーザー名 '{{.username}}' は無効です",
+	"User name must be 60 chars or less.": "ユーザー名は 60 文字以内でなければなりません。",
+	"Userspace file server is shutdown": "ユーザースペースのファイルサーバーが停止しました",
+	"Userspace file server:": "ユーザースペースのファイルサーバー:",
+	"Userspace file server: ": "ユーザースペースのファイルサーバー: ",
+	"Using image repository {{.name}}": "{{.name}} イメージリポジトリーを使用しています",
+	"Using image {{.registry}}{{.image}}": "{{.registry}}{{.image}} イメージを使用しています",
+	"Using image {{.registry}}{{.image}} (global image repository)": "{{.registry}}{{.image}} イメージ (グローバルイメージリポジトリー) を使用しています",
+	"Using the '{{.runtime}}' runtime with the 'none' driver is an untested configuration!": "' none' ドライバーでの '{{.runtime}}' ランタイム使用は、未テストの設定です！",
+	"Using the {{.driver}} driver based on existing profile": "プロフィールを元に、 {{.driver}} ドライバーを使用します",
```
[the remark this answers is not in the document]
Comment (yosshy) [reply translations/ja.json:812]: 指摘の通りですね。確認ありがとうございます。

File: translations/ja.json (hunk 1 above)
Comment (spowelljr) [reply translations/ja.json]: You can run `make test` and that will look for any mistakes in the translations files.

**Comment by atoato88** [comment]
/lgtm
OK as Japanese review.

**Comment by atoato88** [comment]
@yosshy @t-inu 
BTW, It's OK to add `lgtm` label to PR if contents is OK for Japanese review by each other, I think.

File: translations/ja.json (hunk 1 above)
Comment (yosshy) [reply translations/ja.json]: @spowelljr thank you for your reply. It’s helpful for us.

**Comment by yosshy** [comment]
/assign @spowelljr
It’s also ready. Please merge it.

**Review by spowelljr — APPROVED** [review]

**Comment by yosshy** [comment]
@spowelljr thanks for merging it.

**Pull request merged**